facade/nodeFacade_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/multiversx/mx-chain-go/common"
 	"github.com/multiversx/mx-chain-go/config"
 	"github.com/multiversx/mx-chain-go/debug"
+	globalErrors "github.com/multiversx/mx-chain-go/errors"
```

Comment:
That's the package name we usually use for this
```suggestion
	errorsMx "github.com/multiversx/mx-chain-go/errors"
```